### ISSUE TITLE
Encoding troubleshooting

### ISSUE THREAD

**Issue by continue98** [Southclaws/go-samp-query#2]
Hi. samp go query has a bug with Cyrillic and other languages (like Chinese). The easiest way is to look at the language that is installed on the server (ie: for Russian language should be alpha code with [ISO 3166](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes)). Then we can say that for Russian server we should use windows1251. 

Auto-detection works very incorrectly, if you look at different network protocols (for example imap), you will see that the server gives the resultant encoding.

**If you like that solution, I canbe to fix it.**

@Southclaws

**Comment by continue98** [comment]
samp go query causes problems in the master list of servers in open.mp. When Russian works well in SAMP, it works badly in omp and the reason is go samp query

**Comment by Southclaws** [comment]
Yeah this has been a problem for a while - currently we guess the encoding using chardet but it's not perfect.

There are likely some heuristics that are sa-mp specific we can use to do this in a basic way - even checking if the gamemode/hostname contains common phrases like "ru" or other common byte sequences.

PRs welcome, I can help review solutions!
